consensus/enclave/mock/src/lib.rs

```diff
@@ -32,19 +33,37 @@ use mc_transaction_core::{
 use mc_util_from_random::FromRandom;
 use rand_core::SeedableRng;
 use rand_hc::Hc128Rng;
-use std::{convert::TryFrom, sync::Arc};
+use std::{
+    convert::TryFrom,
+    sync::{
+        atomic::{AtomicU64, Ordering},
+        Arc,
+    },
+};
 
-#[derive(Clone)]
 pub struct ConsensusServiceMockEnclave {
     pub signing_keypair: Arc<Ed25519Pair>,
+    pub minimum_fee: AtomicU64,
+}
+
+impl Clone for ConsensusServiceMockEnclave {
+    fn clone(&self) -> Self {
+        Self {
+            signing_keypair: self.signing_keypair.clone(),
+            minimum_fee: self.minimum_fee.load(Ordering::Acquire).into(),
```

Comment:
Why do you not want this to behave similar to `signing_pair` where the underlying instance is shared between the clones? I think it should be wrapped in an `Arc` similarly to `signing_keypair`.